src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -144,6 +145,10 @@ cd "$SOURCE_DIR"
 {{ end }}
 
 {{ if RunNpmPack }}
+	{{ if PackageDirectory | IsNotBlank }}
+	echo "Switching to package directory provided: '{{ PackageDirectory }}'..."
+	cd "$SOURCE_DIR/$packageDirName"
```

Comment:
we should check if the $packageDirName i.e. the packagedir exists, if not we should throw error